lib/cc/analyzer/container.rb

```diff
@@ -72,6 +72,8 @@ def run(options = [])
           @listener.finished(container_data(duration: duration, status: status))
         end
 
+        p status
```

Comment:
Oh I'm sorry. I started down a debugging track on Circle with WIP commits. I should've marked the PR itself WIP too.